### DISCUSSION TITLE
我自己整理了一下之前的CCRD数据库数据，是否上传到搜索引擎？

### DISCUSSION THREAD

**Discussion by CiangCing14** [banned-historical-archives/banned-historical-archives.github.io#3950] [General]
https://ccrd2024.github.io/
https://github.com/ccrd2024/ccrd2024.github.io
（预计本文发表后过几个小时网站的编译就完成了。）
我已经做好了sitemap，是通过自制代码生成的sitemap。
[generate_sitemap.py.zip](https://github.com/banned-historical-archives/banned-historical-archives.github.io/files/14528697/generate_sitemap.py.zip)
请问把sitemap上传到搜索引擎是否会牺牲网站的安全性？是否会可能减少网站寿命？那么还是要选择不上传到搜索引擎，以保护网站安全性和寿命为先？请问你们怎么看？

**Comment by CiangCing14** [comment]
刚才文件大小超限了，我改了一下内容，应该会通过编译。

**Comment by CiangCing14** [comment]
网站已经编译完成，现在看起来还可以。

**Comment by banned-historical-archives** [comment]
不影响安全性

**Comment by banned-historical-archives** [comment]
搜索引擎录入应该会增加网站寿命，可能会被 archive.org 发现并永久收录
